manifests/03-rbac-role-cluster.yaml

```diff
@@ -80,6 +80,9 @@ rules:
   - create
   - update
   - delete
+- apiGroups: [""]
```

Comment:
This is required to run `helm list --all-namespaces` with Helm SDK. 

Error without it: `metric helm_chart_release_health_status unhandled: list: failed to list: secrets is forbidden: User "system:serviceaccount:openshift-console-operator:console-operator" cannot list resource "secrets" in API group "" at the cluster scope`